### ISSUE TITLE
Provider "iam" not available for installation.

### ISSUE THREAD

**Issue by VidyasagarMSC** [IBM-Cloud/terraform-provider-ibm#397]
Hi there,
I am trying to create an access group and access group policy using v0.12.0 of the IBM cloud provider and seeing this error

> Initializing provider plugins...
> - Checking for available provider plugins on https://releases.hashicorp.com...
> 
> Provider "iam" not available for installation.
> 
> A provider named "iam" could not be found in the official repository.
> 
> This may result from mistyping the provider name, or the given provider may
> be a third-party provider that cannot be installed automatically.
> 
> In the latter case, the plugin must be installed manually by locating and
> downloading a suitable distribution package and placing the plugin's executable
> file in the following directory:
>     terraform.d/plugins/darwin_amd64
> 
> Terraform detects necessary plugins by inspecting the configuration and state.
> To view the provider versions requested by each module, run
> "terraform providers".

### Terraform Version

Terraform v0.11.8
+ provider.ibm (unversioned)
+ provider.local v1.1.0

**Terraform providers**
.
├── provider.iam
├── provider.ibm
├── provider.local
└── provider.terraform

### Expected Behavior
Should create an iam_accessgroup and policy

### Actual Behavior
Throwing an error

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform init`

**Comment by Praveengostu** [comment]
Hi @VidyasagarMSC We have not released any provider with name `iam`. Unders the IBM Cloud provider we had added new resources for iam_access_group(https://ibm-cloud.github.io/tf-ibm-docs/v0.12.0/r/iam_access_group.html) and iam_access_group_policy(https://ibm-cloud.github.io/tf-ibm-docs/v0.12.0/r/iam_access_group_policy.html) . There are example usage as well.

**Comment by VidyasagarMSC** [comment]
@Praveengostu Thanks for the prompt reply. Using the same examples in `main.tf` is throwing the above error while executing `terraform init`

**Comment by VidyasagarMSC** [comment]
@Praveengostu Looks like a documentation error, `ibm_` is missing while creating the access group in this link 
https://ibm-cloud.github.io/tf-ibm-docs/v0.12.0/r/iam_access_group_policy.html

> 
> resource "iam_access_group" "accgrp" {
>   name = "test"
> }
> 
> resource "ibm_iam_access_group_policy" "policy" {
>   access_group_id = "${iam_access_group.accgrp.id}"
>   roles        = ["Viewer"]
> }
>

**Comment by Praveengostu** [comment]
@VidyasagarMSC Yes We will fix the documentation.. Hope after prefixing with `ibm_` you dont have any issue.
